client/webpack.config.js

```diff
@@ -355,6 +376,18 @@ const webpackConfig = {
 		shouldProfile && new webpack.ProgressPlugin( { profile: true } ),
 
 		shouldUsePersistentCache && shouldUseReadonlyCache && new ReadOnlyCachePlugin(),
+
+		// NOTE: Must be last.
```

Comment:
Could you add a bit more context to this comment? Last as in what?

Reply:
Good point. Sentry docs say their webpack plugin needs to be the last registered plugin in Webpack so that it can see all the changes made by other plugins.